### PR TITLE
adding the KCP-Edge project

### PR DESCRIPTION
**Pull request by clubanderson**
thank you for considering our project for inclusion

### CODE REVIEW

**Comment by clubanderson** [comment]
@shiftkey can you approve?

**Comment by clubanderson** [comment]
@ritwik12 can you approve?

File: _data/projects/kcp-edge.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: KCP-Edge is a subproject of kcp (a small, Kubernetes-like control plane for workloads on many clusters) focusing on concerns arising from edge multicloud use cases and the difficulties associated with using Kubernetes to address them.
+site: https://github.com/kcp-dev/edge-mc/
+tags:
+  - kubernetes
+  - container
+  - orchestration
+  - devops
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/kcp-dev/edge-mc/contribute
```
Comment (ritwik12) [inline _data/projects/kcp-edge.yml]: This needs to be a link to the label you have mentioned above

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/kcp-edge.yml (hunk 1 above)
Comment (clubanderson) [reply _data/projects/kcp-edge.yml]: in latest commit, 'link' and 'site' now match.  Is that what you are asking for?

Comment (clubanderson) [reply _data/projects/kcp-edge.yml]: @ritwik12 is this the correction you requested?

File: _data/projects/kcp-edge.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+desc: KCP-Edge is a subproject of kcp (a small, Kubernetes-like control plane for workloads on many clusters) focusing on concerns arising from edge multicloud use cases and the difficulties associated with using Kubernetes to address them.
+site: https://github.com/kcp-dev/edge-mc/
+tags:
+  - kubernetes
+  - container
+  - orchestration
+  - devops
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/kcp-dev/edge-mc/
```
Comment (ritwik12) [inline _data/projects/kcp-edge.yml]:
```suggestion
  link: https://github.com/kcp-dev/edge-mc/labels/good%20first%20issue
```
You need to create a label in your repo "up for grabs" or use some other existing label such as "good first issue"

**Review by ritwik12 — DISMISSED** [review]

**Comment by clubanderson** [comment]
We already have "good first issue". @ritwik12

**Comment by clubanderson** [comment]
committed your suggestion, Thank you @ritwik12

**Comment by clubanderson** [comment]
@ritwik12 can you approve?

**Review by shiftkey — APPROVED** [review]

**Comment by clubanderson** [comment]
@shiftkey thank you, will this merge now?

**Pull request merged**